README.md

```diff
@@ -17,6 +17,7 @@
     2. Toss ball vertically upwards, without imparting spin, so that it rises at least 16cm (approximately 6in)
 3. If a player bests their opponent by three consecutive games, they are required to offer a 'winner takes all' round. The winner of the 'winner takes all' round becomes the *CHAMPION*.
 4. If during any championship match a player is beaten 11-0, skunk rules go into effect which automatically make the winning player the *CHAMPION*.
+5. If a player loses 3 matches in a row to the defending *CHAMPION*, they lose their privalege to challenge until a new *CHAMPION* has been crowned. 
```

Comment:
`privalege` to `privilege`.
Also I don't like this rule.

Reply:
i suppose we need a vote

Reply:
I updated spelling. Rule state we need a 2/3 majority. But who are the voting member?

Reply:
```
In order to be a voting member, a player must meet one of the following requirements:
- Contributing member to the ping-pong-champion-rules git repo.
- Won a championship title at some time in the past.
- A current Vivid Seats employee that has seen the movie Balls of Fury
```

Reply:
my vote is, 👎

Reply:
so pretty much anyone in #pongers-whiff-whaff